sale_promotion_rule/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com)
+# Benoît GUILLOT <benoit.guillot@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Promotion Rule",
+    "summary": "Module to manage promotion rule on sale order",
+    "version": "10.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Sale",
+    "website": "https://akretion.com",
+    'author': 'Anybox, ACSONE SA/NV,Odoo Community Association (OCA)',
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
+        "python": [],
+        "bin": [],
+    },
+    "depends": [
+        "sale",
+    ],
+    "data": [
```

Comment:
sequence as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

Reply:
sequence ?????? What's the problem here? The order of data files is not enforced by the template. The most important IMO is that data files are grouped by type.  I don't plan o change this part of the manifest.